src/otx/algorithms/detection/configs/base/data/semisl/base_semisl_det_data_pipeline.py

```diff
@@ -8,18 +8,13 @@
 # This is from otx/mpa/recipes/stages/_base_/data/pipelines/ubt.py
 # This could be needed sync with incr-learning's data pipeline
 __img_scale = (992, 736)
+
 __img_norm_cfg = dict(mean=[0, 0, 0], std=[255, 255, 255], to_rgb=True)
 
 common_pipeline = [
     dict(
         type="Resize",
-        img_scale=[
-            (992, 736),
-            (896, 736),
-            (1088, 736),
-            (992, 672),
-            (992, 800),
-        ],
+        img_scale=__img_scale,
```

Comment:
Is this change somehow beneficial for all SSL detection models?

Reply:
Let's revert it for now